### PR TITLE
Finish renaming MPAS components and file output

### PR DESCRIPTION
**Pull request by jonbob**
DO NOT MERGE - still testing

This PR continues the process of renaming the MPAS components and the component output files to match E3SM naming conventions. It completes the transition of renaming mpas-cice to mpas-seaice (mpassi as a short name). It also modifies the scripts that create MPAS component streams files to match the naming convention of the other E3SM components, with the casename prepended to all output filenames.

Tested with:
* ERS.ne4_oQU240.A_WCYCL1850.anvil_intel
* ERS.T62_oQU120_ais20.MPAS_LISIO_TEST.anvil_intel

### CODE REVIEW

**Comment by jonbob** [comment]
@mark-petersen - no, CICE is still a component in E3SM so it's not quite that easy. Plus:
* the svn repo still has mpas-cice subdirectory for the inputdata files;
* the old graph files are still mpas-cice.graph.info for similar reasons; and
* there are a couple of other input files with mpas-cice in their names.
But getting close...

**Comment by jonbob** [comment]
@jgfouca - whenever you have a chance, I removed a section of code in cime/scripts/lib/CIME/case/case_st_archive.py that treated the MPAS filenames specially. If you could give that an extra look I'd very much appreciate it. Plus there's a similar file with a .pyc suffix that I don't know about...

**Comment by jonbob** [comment]
@xylar - could you please look this over at some point and make sure it won't create untoward problems in the analysis system? Thanks

File: cime/scripts/lib/CIME/case/case_st_archive.py [hunk 1]
```diff
@@ -390,21 +390,16 @@ def _archive_restarts_date_comp(case, archive, archive_entry,
     for suffix in archive.get_rest_file_extensions(archive_entry):
         for i in range(ninst):
             restfiles = ""
-            if compname.find("mpas") == 0:
-                pattern = compname + r'\.' + suffix + r'\.' + '_'.join(datename_str.rsplit('-', 1))
-                pfile = re.compile(pattern)
-                restfiles = [f for f in os.listdir(rundir) if pfile.search(f)]
+            pattern = r"{}.{}[\d_]*\..*".format(casename, compname)
```
Comment (jgfouca) [inline cime/scripts/lib/CIME/case/case_st_archive.py]: Thanks for removing the special case for mpas!

**Comment by jonbob** [comment]
Thanks @jgfouca - I wasn't sure if I needed to do anything with that one. Good to know

**Comment by rljacob** [comment]
I was thinking about the input data dir and we can probably just rename "mpas-cice" to "mpassi" and then set up a soft link.

**Comment by jonbob** [comment]
@rljacob - I like the idea of changing the svn repo to match names. Any sense of how difficult that would be? The high-level ice subdirectory, but we also have a bunch of decomposition files that have names like "mpas-cice.graph.info.170905.part.960". I'm happy to do the legwork if you can remember svn well enough....

**Comment by rljacob** [comment]
Renaming things in svn is a pain so I would instead delete the entire mpas-cice from https://acme-svn2.ornl.gov/acme-repo/acme/inputdata/ and rely on the copy on blues which is served by http and is just a regular directory structure.  Then we can just unix rename the directory.

After that, in the blues inputdata server:  make a copy of every mpas-cice.* file with the new name and update the date string.  Then make a PR that changes all internal references to use the new names.  Once that is merged to master, you can delete the old names from the input data server.  Then go around the various local inputdata dirs and remove all the "mpas-cice" files.

Don't delay this PR for that.

**Comment by rljacob** [comment]
Wait we don't want to delete the old files because old versions will need them.  So just make a new directory on blues for mpas-seaice, populate it with files with the right names and then make a PR to use those new names.

**Comment by jonbob** [comment]
Thanks @rljacob - good call

**Comment by rljacob** [comment]
Is this PR still being tested?

**Comment by jonbob** [comment]
It's just about ready -- probably Monday, if that fits in the schedule. We're mostly trying to get the inputdata repo consistent, and make sure those name changes are better thought out

**Comment by jgfouca** [comment]
@jonbob , any updates? I'm waiting for this to go in before I do a e3sm->cime split.

**Comment by jonbob** [comment]
@jgfouca - it's ready but will fail all tests will seaice as long as the anvil server is down. On my tests, the inputdata download fails with an error from the lcrc server

**Comment by jonbob** [comment]
Passes:
* SMS_D.T62_oQU240.GMPAS-IAF.edison_intel
* SMS_D.T62_oQU120_ais20.MPAS_LISIO_TEST.edison_intel
but only when the order of servers is changed in config_inputdata.xml

**Comment by susburrows** [comment]
@sterlingbaldwin FYI: this pending PR might affect the processflow tool (by renaming more of the mpas-cice output)

**Comment by sterlingbaldwin** [comment]
Is there any output data thats been created using this branch I could use for testing purposes?

**Comment by rljacob** [comment]
@jonbob ready to resume merging this PR?

**Comment by jonbob** [comment]
@rljacob - yes, though I'm guessing it will be a semi-painful rebase. I'll start working on it next week.

**Comment by jonbob** [comment]
I hope to soon, but I'm buried right now

**Comment by rljacob** [comment]
From IG telecon: this will have to be redone on a new branch so closing.

**Pull request closed without merging**